common/test/acceptance/fixtures/course.py

```diff
@@ -222,6 +222,26 @@ def configure_course(self):
         """
         self._configure_course()
 
+    def get_course_outline(self):
```

Comment:
Why have this get method in the course object versus the properties below?  If we're asking the object to return one of its attributes/values, then a property makes more sense to me.